### ISSUE TITLE
(WCAG) Missing attribute in the validation message role="alert"

### ISSUE THREAD

**Issue by amerimohammed** [bsgriggs/mendix9-searchable-reference-selector#27]
For screen readers to detect the validation messages role="alert" must be one of the attributes as it is the case in the default Mendix validation messages.

**Comment by bsgriggs** [comment]
released in version 6.3.2
